corehq/apps/app_manager/helpers/validators.py

```diff
@@ -745,9 +746,14 @@ def validate_for_build(self, validate_module):
                     self.form.get_app().get_form(form_link.form_id)
                 except FormNotFoundException:
                     errors.append(dict(type='bad form link', **meta))
+        elif self.form.post_form_workflow == WORKFLOW_MODULE:
+            if module.put_in_root:
+                errors.append(dict(type='form link to display only forms', **meta))
```

Comment:
Out of curiosity, why is this not allowed?

Reply:
It doesn't work because there's no menu screen to go to - say the module is m0, there's no `<menu id="m0">` in the suite. So it just sends you to the home screen.